web/src/lib/components/elements/date-input.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+  import type { HTMLInputAttributes } from 'svelte/elements';
+
+  interface $$Props extends HTMLInputAttributes {
+    type: 'date' | 'datetime-local';
+  }
+
+  export let value: $$Props['value'] = undefined;
```

Comment:
This is cool! Didn't know you could do this in svelte 4 already - I was excited for this in svelte 5 though haha